ubuntu2004_cuda/Dockerfile

```diff
@@ -53,7 +53,7 @@ ENV PREFIX /usr/local
 # CMake 3.17.3 version in APT is too old
```

Comment:
We may want to correct this typo at one point. Ubuntu 20.04 comes with CMake 3.16.3 out of the box. If it came with 3.17.3, this whole piece of code would not have been necessary. :stuck_out_tongue: